examples/aides_logement/dune

```diff
@@ -0,0 +1,33 @@
+(rule
+ (targets aides_logement.ml aides_logement_api_web.ml)
+ (deps
+  (source_tree ..)
+  ../../compiler/plugins/api_web%{ext_plugin})
+ (action
+  (run
+   catala
+   api_web
+   -t
+   --plugin-dir=../../compiler/plugins
+   aides_logement.catala_fr)))
+
+(rule
+ (targets aides_logement_schema.json)
+ (deps
+  (source_tree ..)
+  ../../compiler/plugins/json_schema%{ext_plugin})
```

Comment:
Ok we should leave some documentation about how to write those kind of rules in `examples/README.md`

Reply:
Hm indeed! Note that we don't have dependency analysis here (shouldn't we extend Clerk ?) so I include the whole `example` subtree.

Reply:
There is a `Makefile` backend to Catala that performs dependency analysis (inclusions) when called on a file, it produces a `.d` file.

Reply:
I know ! But there is no straight-forward way to use those in dune (that I know about; support for `ocamldep` is built-in) — so I opted for the lazy solution for now.

Reply:
Ok, we can also add a new backend to the Catala compiler, who would share a lot of code with the Makefile backend, that would produce dune rules exactly as we expect.